src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -15,6 +15,7 @@ def write_subject_to_nwb(
     individual_name: str,
     config_file: FilePathType,
     timestamps: Optional[Union[List, np.ndarray]] = None,
+    name: Optional[str] = "PoseEstimation",
```

Comment:
```suggestion
    container_name: Optional[str] = "PoseEstimation",
```

Would probably want to be a little more explicit since this technically makes a container of many neurodata objects